src/components/ui/sidebar/facility-nav.tsx

```diff
@@ -67,16 +68,34 @@ function generateFacilityLinks(
         {
           name: t("search_patients"),
           url: `${baseUrl}/patients`,
+          matchPaths: [`${baseUrl}/patients/verify`],
         },
         {
           name: t("encounters"),
           url: `${baseUrl}/encounters/patients`,
+          matchPaths: [
+            `${baseUrl}/patient/:patientId/encounter/:encounterId/updates`,
```

Comment:
Hmm, hard coding isn't scalable, since plugs add tabs for encounters (check EncounterShow).

Reply:
@sindhureddy-6 ☝️

Reply:
Also fix lint issues for deployment 👍